src/style.css

```diff
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
point 5, the same at all tasks
https://mateacademy.slack.com/archives/C02LA8ED589/p1637236987052200